docs/examples/storage/object/variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "tenancy_ocid" {}
+variable "user_ocid" {}
+variable "fingerprint" {}
+variable "private_key_path" {}
+variable "namespace_name" {}
+variable "compartment_ocid" {}
+
+variable "BucketName" {
+    default = "BucketOne"
+}
```

Comment:
Defining this variable only sort of complicates the example, guessing it was just a test